Src/Sarif.PatternMatcher.Cli/ExportSearchDefinitionsCommand.cs

```diff
@@ -0,0 +1,363 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
```

Comment:
What is this file about? A high level comment would help.

Reply:
this is a command line option that would  be able to parse a few formats in searchDefinitions. The searchDefinitions are used to generate the rules.

Reply:
Thanks Eddy.